payment_source.go

```diff
@@ -38,6 +38,7 @@ type SourceVerifyParams struct {
 	Params
 	Customer string
 	Amounts  [2]uint8
+	Values   []string
```

Comment:
Would you mind commenting `Amounts` here to indicate that it's for bank accounts only and essentially deprecated.